src/rp2040/main.c

```diff
@@ -144,7 +144,8 @@ clock_setup(void)
     watchdog_hw->tick = 1 | WATCHDOG_TICK_ENABLE_BITS;
 
     // Enable GPIO control
-    enable_pclock(RESETS_RESET_IO_BANK0_BITS | RESETS_RESET_PADS_BANK0_BITS);
+    enable_pclock(RESETS_RESET_IO_BANK0_BITS | RESETS_RESET_PADS_BANK0_BITS
+                  | RESETS_RESET_PWM_BITS);
```

Comment:
I think it would be preferable to enable the PWM clock in hard_pwm.c (as this follows the common behaviour on rp2040 and other architectures).